server/models/User.js

```diff
@@ -1,21 +1,21 @@
 const mongoose = require("mongoose");
 const bcrypt = require("bcryptjs");
 
-const userSchema = new mongoose.Schema({
-  email: {
-    type: String,
-    required: true,
-    unique: true,
+const userSchema = new mongoose.Schema(
+  {
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    password: {
+      type: String,
+      required: true,
+      select: false,
+    },
   },
-  password: {
-    type: String,
-    required: true,
-  },
-  registerDate: {
-    type: Date,
-    default: Date.now,
-  },
-});
+  { timestamps: true }
+);
```

Comment:
Should probably have a property of name for users? How would you show collaborators otherwise aside from emails (unless that's fine)

Reply:
I mentioned this in Slack, easy change to make. Will do it when I wire up team boards tonight to 👍🏻